### PR TITLE
Fixes #509 : Implement human body resistance measurement experiment

### LINKED ISSUE #509: Implement Human Body Resistance Measurement experiment
**Actual Behaviour**

Human body resistance measurement experiment is not implemented in saved experiments

**Expected Behaviour**

Experiment needs to be implemented in the saved experiments school level section

**Steps to reproduce it**

1. Open app
2. Go to `Saved Experiments`
3. Select `School Level` and `Human body resistance` experiment

**LogCat for the issue**

> Not available

**Screenshots of the issue**

> Not available

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by CloudyPadmal**
Fixes issue #509 

Changes: 
- Implemented experiment as in [this](https://github.com/fossasia/pslab-desktop-apps/blob/development/psl_res/GUI/Z_SCHOOL_LEVEL/A_voltage_fundamentals/E_RESISTANCE_BODY.py) (pslab-desktop-apps)

Screenshots for the change: 
device-2017-08-14-161221

APK for testing:
[app-debug.apk.zip](https://github.com/fossasia/pslab-android/files/1221626/app-debug.apk.zip)

### CODE REVIEW

**Comment by akarshan96** [comment]
@CloudyPadmal please see what's causing the build to fail

File: app/src/main/java/org/fossasia/pslab/adapters/PerformExperimentAdapter.java [hunk 1]
```diff
@@ -139,11 +142,10 @@ public Fragment getItem(int position) {
                     return LemonCellExperiment.newInstance();
                 if (experimentTitle.equals(context.getString(R.string.capacitor_discharge)))
                     return CapacitorDischargeExperiment.newInstance();
-                if (experimentTitle.equals(context.getResources().getString(R.string.full_wave_rectifier)))
```
Comment (viveksb007) [inline app/src/main/java/org/fossasia/pslab/adapters/PerformExperimentAdapter.java]: you have removed case for full wave rectifier. Please add it again and replace hard-coded strings with string resources in both ```case 0``` and ```case 1```

**Comment by viveksb007** [comment]
@CloudyPadmal Please resolve the merge conflicts.

**Comment by CloudyPadmal** [comment]
Hi @viveksb007 ! I've resolved the conflict! :+1:

**Comment by CloudyPadmal** [comment]
Thanks @viveksb007 ! :+1:

**Pull request merged**